lib/sdn_epc/forwarder.ex

```diff
@@ -0,0 +1,55 @@
+defmodule SdnEpc.Forwarder do
+  use GenServer
+
+  # Client API
+
+  def start_link do
+    res = GenServer.start_link(__MODULE__, :ok, name: __MODULE__)
```

Comment:
The `res` is not needed here. The `start_link` function is simply supposed to return `{:ok, pid}` so the pattern match here is at least redundant.